### PR TITLE
Added handling of versioning using extension.json on the update-script-command

### PR DESCRIPTION
**Pull request by qeude**
## Description

Improving [update-scripts-command](https://github.com/raycast/script-commands/blob/master/commands/system/update-scripts-command.template.py) to handle script versionning using the [file](https://github.com/raycast/script-commands/blob/master/commands/extensions.json) generated by the Toolkit.
It now only updates when needed and the user's feedback about what happened is improved.
See #448 

## Type of change

- [x] Improvement of an existing script

## Checklist

- [x] I have read [Contribution Guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md)

### CODE REVIEW

File: commands/system/update-scripts-command.template.py [hunk 1]
```diff
@@ -7,8 +7,9 @@
 
 # Required parameters:
 # @raycast.schemaVersion 1
-# @raycast.title Update Community Scripts
-# @raycast.mode inline
+# @raycast.title Update Community Scripts 
+# @raycast.description Updates community scripts to their last available version from the Github repository.
```
Comment (unnamedd) [inline commands/system/update-scripts-command.template.py:11]:
```suggestion
# @raycast.description Updates community Script Commands to their last available version from the GitHub repository.
```

**Review by unnamedd — COMMENTED** [review]
Thanks for the improvement @qeude! Nice to see your Script Command making use of the `extensions.json`.

**Comment by grzegorzkrukowski** [comment]
Wow - really quick and nice update.

I am thinking of one last thing: do we need a `.template` file here or could we just use script current directory `./` as default and skip the manual setup?

**Comment by qeude** [comment]
@grzegorzkrukowski I'm not sure it would be a good idea since the scripts isn't located in your scripts root dir for sure that's why I made a ˋ.template`

**Comment by grzegorzkrukowski** [comment]
Ok merging it as it is then !

**Pull request merged**